src/client/api/mgmt.c

```diff
@@ -22,32 +22,78 @@
  */
 #define D_LOGFAC	DD_FAC(client)
 
+#include <sys/wait.h>
 #include <daos/mgmt.h>
 #include <daos/pool.h>
 #include <daos/task.h>
 #include <daos_mgmt.h>
 #include "client_internal.h"
 #include "task_internal.h"
 
+static int
+daos_dmg_fork_exec(char *argv[])
+{
+	pid_t pid = fork();
+
+	if (pid == -1) {
+		D_ERROR("fork() failed to create child\n");
+		return -DER_INVAL;
+	} else if (pid == 0) {
+		/* child process */
+		D_INFO("child pid = %d\n", getpid());
+		execvp(argv[0], argv);
+		exit(DER_SUCCESS);
```

Comment:
execvp should not return.  If we get here, something went wrong and we should return an error.